bliss/catalog.py

```diff
@@ -722,3 +722,35 @@ def to_astropy_table(self, encoder_survey_bands: Tuple[str]) -> Table:
         est_cat_table.remove_column("galaxy_params")
 
         return hstack([est_cat_table, galaxy_params_table])
+
+    def filter_full_catalog_by_ploc_box(self, box_origin: torch.Tensor, box_len: float):
```

Comment:
Could you add a unit test of this function to `test_catalogs.py`?

Reply:
I wrote a new test `test_filter_full_catalog_by_ploc_box` for it